File: 프로그래머스/unrated/181905. 문자열 뒤집기/문자열 뒤집기.js
function solution(my_string, s, e) {
     let params = my_string
    .substring(s, e + 1)
    .split("")
    .reverse();

  let strArr = my_string.split("");

  strArr.splice(s, e - s + 1, ...params);

  return strArr.join("");
}

File: 프로그래머스/unrated/181905. 문자열 뒤집기/문자열 뒤집기.java
class Solution {
    public String solution(String my_string, int s, int e) {
        String[] strArr = my_string.split( "" );
		String[] params = my_string.substring( s, e + 1 )
								   .split( "" );
		int temp = 0;
		for ( int i = 0; i < strArr.length; i++ ) {
			if ( temp > params.length - 1 )
				break;
			if ( i >= s && i <= e ) {
				strArr[ i ] = params[ params.length - 1 - temp ];
				temp++;
			}
		}
		return String.join( "", strArr );
    }
}